_includes/custom-head.html

```diff
@@ -7,18 +7,18 @@
 
 
 {%- include favicons.html -%}
-<link href="https://unpkg.com/@primer/css/dist/primer.css" rel="stylesheet" />
-<link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+<link href="https://cdn.jsdelivr.net/npm/@primer/css/dist/primer.css" rel="stylesheet" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
```

Comment:
![image](https://user-images.githubusercontent.com/1483922/93914998-387d2c00-fcbc-11ea-82e1-85e37130ba44.png)

Reply:
Not able to access this newer CDN

Reply:
It seems that there is no way to verify sha-256 in the third-party CDN, I changed back to the previous version

Reply:
We hope to find a suitable solution for Chinese users.